src/index.ts

```diff
@@ -99,16 +100,22 @@ async function validateArtifact(
  * Downloads an artifact from an Electron release and returns an absolute path
  * to the downloaded file.
  *
+ * Each release of Electron comes with artifacts, many of which are
+ * platform/arch-specific (e.g. `ffmpeg-v31.0.0-darwin-arm64.zip`) and others that
+ * are generic (e.g. `SHASUMS256.txt`).
+ *
+ *
  * @param artifactDetails - The information required to download the artifact
+ * @category Download Artifact
  */
 export async function downloadArtifact(
-  _artifactDetails: ElectronPlatformArtifactDetailsWithDefaults,
+  artifactDetails: ElectronPlatformArtifactDetailsWithDefaults | ElectronGenericArtifactDetails,
```

Comment:
Doesn't make sense to have the function parameter start with `_` so I changed the variable naming around.